db/seeds/development/7_groups.rb

```diff
@@ -0,0 +1,20 @@
+@alphabet = ("a".."z").to_a
```

Comment:
Don't need `@` here, we only need `@` if we're passing something down to a view or if it's a instance variable.